README.md

```diff
@@ -0,0 +1,159 @@
+
+## Installation
+
+TK
+
+## Widget options
+
+The `example` project in this repo contains the following sample form implementation:
+
+```python
+class ExampleGeoMultipleChoiceForm(forms.ModelForm):
+    class Meta:
+        model = SelectedArea
+        fields = '__all__'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        settings_overrides = {
+            'DEFAULT_ZOOM': 12,
+            'DEFAULT_CENTER': (41.88, -87.7),
+            'RESET_VIEW' : True, # Defaults to True
+            # Sets the bounds of RESET VIEW; y min, x min, y max, x max
+            # See https://github.com/makinacorpus/django-leaflet/issues/192
+            'SPATIAL_EXTENT': (-87.3, 41.5, -88, 42.15),
+            'MAP_ID': 'my-example-map', # Defaults to 'map'
+            'MAP_HEIGHT': '400px',
+            'MAP_WIDTH': '100%',
+            'MAP_LAYER_STYLE': {
+              'color': '#7a7a7a',
+              'weight': 3,
+              'opacity': 0.5,
+              'fillColor': '#999999',
+              'fillOpacity': 0.3,
+            },
+            'MAP_LAYER_SELECTED_STYLE': {
+              'color': '#7a7a7a',
+              'weight': 3,
+              'opacity': 0.5,
+              'fillColor': 'black',
+              'fillOpacity': 0.7
+            }
+        }
+
+        self.fields['areas'].widget = GeoMultipleChoiceWidget(
+            choices=[
+                (choice.id, choice) for choice
+                in Area.objects.all()
+            ],
+            settings_overrides=settings_overrides
+        )
+```
+
+`settings_overrides` accepts all arguments used by `django-leaflet` in its [LEAFLET_CONFIG](https://django-leaflet.readthedocs.io/en/latest/templates.html#configuration), in addition to the following:
+
+- `MAP_ID` sets the html `id` for your widget map. Useful if you have multiple maps on the page.
+- `MAP_HEIGHT` and `MAP_WIDTH` allow you to size the widget in your form. Must be used together.
+- This example uses the default values for `MAP_LAYER_STYLE` and `MAP_LAYER_STYLING`.
+
+`SelectedArea` is a model defined in `example/models.py`, and can be adjusted for your needs.
+
+## Importing Census data
+
+This repo includes an `example` application using geographical data for 2018 Census block groups in Chicago. Look at `data/Makefile` and `management/commands/import_data.py` to see the ETL pipeline for that data.
+
+If you'd like to use the same pipeline in your project but for a different area:
+
+1. Copy the `data` directory into your project
+2. Update `STATES` in `scripts/states.py`. If the area is in a new state, create a new entry in STATES for that state; otherwise, update the existing state.
+3. Update `all` in `data/Makefile` to match the form `$(DATA_DIR)/final/cb_2018_{state id}_bg_500k.shp` for each state you're importing shapefiles for.
+4. To delete and regenerate the shapefiles, run:
+
+```bash
+cd data && make clean && make
+```
+
+Or if your app is containerized:
```

Comment:
Definitely open to changing the way I've handled this. Maybe just not including the specific commands? Listing options here feels messy.

Reply:
I actually like offering the commands in both bare and containerized form, particularly because the average developer may not have as much Make and Docker knowledge as we do.